Makefile

```diff
@@ -50,7 +50,7 @@ endef
 # Targets that are exposed to developers and users.
 build: .format fusedev virtiofs
 release: .format .release_version fusedev virtiofs
-static-release: .musl_target .format .release_version fusedev virtiofs
+static-release: .musl_target .format .release_version fusedev
```

Comment:
`static-release` should include both `fusedev` and `virtiofs` targets. To only build fusedev，please add `fusedev-static-release` target instead.